lhotse/recipes/voxconverse.py

```diff
@@ -0,0 +1,149 @@
+"""
+
+import json
+import logging
+import re
+import shutil
+import zipfile
+from collections import defaultdict
+from concurrent.futures import ProcessPoolExecutor
+from pathlib import Path
+from typing import Dict, Iterable, Optional, Union
+from urllib.error import HTTPError
+
+from tqdm.auto import tqdm
+
+from lhotse import fix_manifests, validate_recordings_and_supervisions
+from lhotse.audio import Recording, RecordingSet
+from lhotse.supervision import AlignmentItem, SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike, is_module_available, resumable_download
+
+DEV_AUDIO_ZIP = (
+    "https://www.robots.ox.ac.uk/~vgg/data/voxconverse/data/voxconverse_dev_wav.zip"
+)
+TEST_AUDIO_ZIP = (
+    "https://www.robots.ox.ac.uk/~vgg/data/voxconverse/data/voxconverse_test_wav.zip"
+)
+ANNOTATIONS_ZIP = "https://github.com/joonson/voxconverse/archive/master.zip"
+
+
+def download_voxconverse(
+    corpus_dir: Pathlike,
+    force_download: bool = False,
+):
+    corpus_dir = Path(corpus_dir)
+    corpus_dir.mkdir(parents=True, exist_ok=True)
+    completed_detector = corpus_dir / ".completed"
+
+    if not completed_detector.is_file() or force_download:
+        print("Downloading VoxConverse dev set")
+        resumable_download(DEV_AUDIO_ZIP, corpus_dir / "dev.zip")
+        with zipfile.ZipFile(corpus_dir / "dev.zip") as zip_f:
+            zip_f.extractall(corpus_dir / "dev")
+
+        shutil.copytree(
+            corpus_dir / "dev/audio", corpus_dir / "dev", dirs_exist_ok=True
+        )
+        shutil.rmtree(corpus_dir / "dev/audio")
+
+        print("Downloading VoxConverse test set")
+        resumable_download(TEST_AUDIO_ZIP, corpus_dir / "test.zip")
+        with zipfile.ZipFile(corpus_dir / "test.zip") as zip_f:
+            zip_f.extractall(corpus_dir / "test")
+
+        shutil.copytree(
+            corpus_dir / "test/voxconverse_test_wav",
+            corpus_dir / "test",
+            dirs_exist_ok=True,
+        )
+        shutil.rmtree(corpus_dir / "test/voxconverse_test_wav")
+
+        print("Downloading VoxConverse annotations")
+        resumable_download(ANNOTATIONS_ZIP, corpus_dir / "annotations.zip")
+        with zipfile.ZipFile(corpus_dir / "annotations.zip") as zip_f:
+            zip_f.extractall(corpus_dir)
+
+        shutil.copytree(
+            corpus_dir / "voxconverse-master", corpus_dir, dirs_exist_ok=True
+        )
+        shutil.rmtree(corpus_dir / "voxconverse-master")
+
+        # cleanup
+        (corpus_dir / "dev.zip").unlink()
+        (corpus_dir / "test.zip").unlink()
+        (corpus_dir / "annotations.zip").unlink()
+        completed_detector.touch()
+
+    print("Done")
+
+
+def prepare_voxconverse(
+    corpus_dir: Pathlike,
+    output_dir: Optional[Pathlike] = None,
+    split_test: bool = True,  # test part is larger than dev part - split it into dev and test by default
```

Comment:
Is this a standard thing to do with this dataset? If not, it would be better to return the splits as defined by the creators by default.